src/observability/summarizer.go

```diff
@@ -165,6 +186,28 @@ func convertSysLogToSysSummaryMap(syslogs []*pb.Alert) {
 			sysSummary.Destination = strings.Split(syslog.Resource, " ")[0]
 		}
 
+		if syslog.Operation == types.SumOpTypeSyscall {
+			if existingClustername != syslog.ClusterName {
+				_, services, _, pods, err = cluster.GetAllClusterResources(syslog.ClusterName)
```

Comment:
I do not prefer calling GetAllClusterResources all the time. Are there any alternatives here?

Reply:
I have analysed the cluster package it seems there are no alternative.

Reply:
For now I am keeping it as it is. Since it's throwing error.
```sh
11:32AM ERR cluster/clusterMgmtHandler.go:88 > Error on response.
[ERROR] - Post "http://localhost:8080/cm/api/v1/cluster-management/get-cluster-ids": dial tcp [::1]:8080: connect: connection refused
```

Reply:
I think this is because you are not forwarding the cluster management service locally. Can you try to forward the cluster management service and test it again @yasin-cs-ko-ak